test/models/user_test.rb

```diff
@@ -0,0 +1,15 @@
+require "test_helper"
+
+class UserTest < ActiveSupport::TestCase
+  test "when the user is admin" do
+    user = users(:admin)
+
+    assert user.admin?
+  end
+
+  test "when the user is not admin" do
+    user = users(:with_profile_one)
+
+    assert_equal false, user.admin?
```

Comment:
You can use `refute` here. So, `refute user.admin?`.

Reply:
I didn't see it, I'll fix it next PR, thank you!